hack/tests/scaffolding/e2e-go-scaffold.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -ex
+
+source hack/lib/test_lib.sh
+
+ROOTDIR="$(pwd)"
+# TODO: remove once PR 1566 is merged
+trap_add 'rm -f $ROOTDIR/go.mod' EXIT
+BASEPROJECTDIR="/tmp/go-e2e-scaffold"
+IMAGE_NAME="quay.io/example/memcached-operator:v0.0.1"
+
+rm -rf $BASEPROJECTDIR
+mkdir -p $BASEPROJECTDIR
+
+pushd "$BASEPROJECTDIR"
+go run "$ROOTDIR/hack/tests/scaffolding/scaffold-memcached.go" --local-repo $ROOTDIR --image-name=$IMAGE_NAME --local-image
```

Comment:
Not a huge deal, since I could fix this in the go-mod PR, but `go run` doesn't work well when running from a directory outside the module where the go file being run is.

I solved this in the go-mod PR by running `go build` before switching to the `$BASEPROJECTDIR` directory, and then just running the executable after switching.

Reply:
Ehh.. maybe don't worry about this, as there will be other things to fix anyway.